lang/python/python-pytest/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-pytest
+PKG_VERSION:=5.4.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=pytest
+PKG_HASH:=84dde37075b8805f3d1f392cc47e38a0e59518fb46a431cfdaf7cf1ce805f970
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-pytest
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Python testing framework
+  URL:=https://docs.pytest.org/en/latest/
+  DEPENDS:= \
+	+python3-light \
+	+python3-atomicwrites \
+	+python3-more-itertools \
+	+python3-py \
+	+python3-attrs \
+	+python3-pluggy \
+	+python3-packaging \
+	+python3-wcwidth
```

Comment:
Also suggest to add:
* [python3-decimal](https://github.com/pytest-dev/pytest/blob/5.4.1/src/_pytest/python_api.py#L7)
* [python3-logging](https://github.com/pytest-dev/pytest/blob/5.4.1/src/_pytest/logging.py#L2)
* [python3-urllib](https://github.com/pytest-dev/pytest/blob/5.4.1/src/_pytest/pastebin.py#L71-L72)

Reply:
added, thx